homeassistant/components/airzone_cloud/binary_sensor.py

```diff
@@ -28,6 +35,19 @@ class AirzoneBinarySensorEntityDescription(BinarySensorEntityDescription):
     attributes: dict[str, str] | None = None
 
 
+SYSTEM_BINARY_SENSOR_TYPES: Final[tuple[AirzoneBinarySensorEntityDescription, ...]] = (
+    AirzoneBinarySensorEntityDescription(
+        attributes={
+            "errors": AZD_ERRORS,
+            "warnings": AZD_WARNINGS,
+        },
```

Comment:
What do these values look like when they are not `None`?

Do these need to be attributes? Can they be as separate sensors?

Reply:
> What do these values look like when they are not `None`?
> 
> Do these need to be attributes? Can they be as separate sensors?

So far I haven't been able to find much documentation about them, but it's similar to the Airzone Local API integration:
https://github.com/home-assistant/core/blob/7a1f0a0b7410590108cb9bd04f80b5cc8a075a86/homeassistant/components/airzone/binary_sensor.py#L41-L49

So far I haven't been able to get any warning but I did get an error one time (when Airzone was updating my system's firmware) and it had the following value:
`["Error 11"]`
https://www.airzonecontrol.com/as/en/support/faqs/my-thermostat-displays-error-11/
Some of these errors seem to be documented in the following link (but unfortunately not all of them):
https://www.airzonecontrol.com/as/en/support/faqs/

Reply:
Why adding something we don't even know what it is?

Reply:
> Why adding something we don't even know what it is?

@gjohansson-ST because it provides information which can be reported to Airzone if a problem occurs on the user's installation.
If there are any warnings or errors, the binary sensor will indicate that is a problem, and the user will be able to check which is the problem thanks to those attributes.

Reply:
I would remove them until we have a valid test case, have documentation, or a valid test case to know for sure what will be in these fields.

Reply:
> I would remove them until we have a valid test case, or have documentation to know for sure what will be in these fields.

Added a test case from the Airzone Web API documentation:
https://developers.airzonecloud.com/docs/web-api/#get-/devices/-deviceId-/status